carbon-design-system/carbon
# Update Icons library URL

Updated in packages/icons-react/README.md non-working Icons library link (https://www.carbondesignsystem.com/guidelines/iconography/library) to working link (https://carbondesignsystem.com/elements/icons/library)

#### Changelog

**Changed**

- Updated in in packages/icons-react/README.md non-working Icons library link (https://www.carbondesignsystem.com/guidelines/iconography/library) to working link (https://carbondesignsystem.com/elements/icons/library)

#### Testing / Reviewing

1. Go to non-working link https://www.carbondesignsystem.com/guidelines/iconography/library and verify that it shows a 404 page
2. Go to working link https://carbondesignsystem.com/elements/icons/library and verify that it displays all of the icons in carbon/icons-react library.
